Registry/RegistryParse.py

```diff
@@ -238,14 +238,31 @@ def __init__(self, buf, offset, parent):
         if _id != 0x66676572:
             raise ParseException("Invalid REGF ID")
 
-        _seq1 = self.unpack_dword(0x4)
-        _seq2 = self.unpack_dword(0x8)
+    def hive_sequence1(self):
+        """
+        Get first sequence number.
+        Believed to be saved when a commit is started.
+        """
+        return self.unpack_dword(0x4)
+
+    def hive_sequence2(self):
+        """
+        Get second sequence number.
+        Believed to be saved to the same value as squence1 when a commit is completed.
+        """
+        return self.unpack_dword(0x8)
 
-        if _seq1 != _seq2:
-            # the registry was not synchronized
-            pass
+    def valid_sequence(self):
```

Comment:
I like to use verbs when the method does something (other than fetch data), so lets make this `validate_sequence_numbers`